tests/EndToEnd/phpcbf_test.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 function tear_down() {
-  rm -r tests/EndToEnd/Fixtures/*.fixed
+  rm -rf tests/EndToEnd/Fixtures/*.fixed
```

Comment:
We don't need `-r` here.

```suggestion
  rm -f tests/EndToEnd/Fixtures/*.fixed
```

Reply:
@fredden Yeah, I kind of had a feeling it wasn't needed, but for now just wanted to fix the E2E tests. Happy to receive any PRs which improve/expand them though ;-)